src/dotnet/APIView/APIViewWeb/Client/src/documentation.ts

```diff
@@ -0,0 +1,41 @@
+﻿$(() => {  
+  const SEL_DOC_CLASS = ".documentation";
+  const SHOW_DOC_CHECKBOX = "#show-documentation-checkbox";
+  const SHOW_DOC_CHECK_COMPONENT = "#show-documentation-component";
+  const SEL_CODE_INNER_CLASS = ".code-inner";
+  const SEL_CODE_LINE = ".code-line";
+
+  $(document).ready(function () {
```

Comment:
You are already in a `document.ready` callback when executing inside `$(() => {... })`